gui/src/components/maintenance/cortx-software.vue

```diff
@@ -47,7 +47,7 @@
           </td>
           <td style="padding-top: 2px;">{{ lastUpgradeStatus.version }}</td>
         </tr>
-        <tr v-if="lastUpgradeStatus.description">
+        <tr v-if="lastUpgradeStatus.description && lastUpgradeStatus.description !== 'unknown_desc'">
```

Comment:
This should be handled at backend and not at frontend. If there is no description in a particular scenario, then description sent should either be empty or not sent at all.

Reply:
As per the discussion, this will be taken care later. Hence approving now.